src/state/slices/common-selectors.ts

```diff
@@ -200,6 +200,37 @@ export const selectAssetsSortedByMarketCapUserCurrencyBalanceAndName =
     },
   )
 
+export const selectAssetsSortedByMarketCapUserCurrencyBalanceCryptoBalanceAndName =
+  createDeepEqualOutputSelector(
+    selectAssets,
+    selectPortfolioAssetBalancesBaseUnit,
+    selectPortfolioUserCurrencyBalances,
+    selectMarketDataUsd,
+    (assets, portfolioCryptoBaseUnitBalances, portfolioUserCurrencyBalances, marketDataUsd) => {
```

Comment:
```suggestion
    (assets, portfolioBalancesCryptoBaseUnit, portfolioBalancesUserCurrency, marketDataUsd) => {
```

Reply:
Changed in [7a160b8](https://github.com/shapeshift/web/pull/8175/commits/7a160b87ffecbd5c7ec949303dcfc169794d066a)